third_party/ibis/ibis_addon/operations.py

```diff
@@ -185,14 +185,16 @@ def sa_format_hashbytes(translator, expr):
 BigQueryExprTranslator._registry[BitXor] = bq_reduction("BIT_XOR")
 BigQueryExprTranslator._registry[Hash] = format_hash_bigquery
 BigQueryExprTranslator._registry[HashBytes] = format_hashbytes_bigquery
+BigQueryExprTranslator._registry[RawSQL] = format_raw_sql
 AlchemyExprTranslator._registry[RawSQL] = format_raw_sql
 AlchemyExprTranslator._registry[HashBytes] = format_hashbytes_alchemy
 MSSQLExprTranslator._registry[HashBytes] = sa_format_hashbytes
+MSSQLExprTranslator._registry[RawSQL] = format_raw_sql
```

Comment:
```suggestion
MSSQLExprTranslator._registry[RawSQL] = sa_format_raw_sql
```

SQL Server uses SQL Alchemy so we need to use the sql alchemy (sa) version of the function